src/core/server/saved_objects/types.ts

```diff
@@ -202,15 +202,10 @@ export interface SavedObjectsType {
    * See {@link SavedObjectsServiceStart.createInternalRepository | createInternalRepository}.
    */
   hidden: boolean;
-  /**
-   * Is the type global (true), or not (false).
-   * @deprecated Use `namespaceType` instead.
-   */
-  namespaceAgnostic?: boolean;
   /**
    * The {@link SavedObjectsNamespaceType | namespace type} for the type.
    */
-  namespaceType?: SavedObjectsNamespaceType;
+  namespaceType: SavedObjectsNamespaceType;
```

Comment:
Before #54605 was merged, the `namespaceAgnostic` field was required. Now that we are removing it, the `namespaceType` field should be required.